### ISSUE TITLE
Create new Aviation site structure taxonomy

### ISSUE THREAD

**Issue by waynedyck** [WSDOT/wsdot-website#508]
Current site structure taxonomy is from CMS. New site structure will be based on suggested IA.

Will need to add new terms to existing taxonomy so pages can be remapped. Once that is complete we can remove the legacy CMS site structure.

**Comment by waynedyck** [comment]
No longer needed as it will be done as part of the new site.
